tests/bootstrap.php

```diff
@@ -35,19 +35,7 @@
 tests_add_filter( 'shutdown', 'drop_tables', 999999 );
 
 require $_tests_dir . '/includes/bootstrap.php';
-require __DIR__ . '/phpunit/traits/base-elementor.php';
-require __DIR__ . '/phpunit/traits/extra-assertions.php';
-require __DIR__ . '/phpunit/traits/auth-helpers.php';
-require __DIR__ . '/phpunit/traits/elementor-library-trait.php';
-require __DIR__ . '/phpunit/traits/breakpoints-trait.php';
-require __DIR__ . '/phpunit/traits/rest-trait.php';
-require __DIR__ . '/phpunit/traits/kit-trait.php';
-require __DIR__ . '/phpunit/base-class.php';
 require __DIR__ . '/phpunit/trait-test-upgrades.php';
-require __DIR__ . '/phpunit/ajax-class.php';
-require __DIR__ . '/phpunit/base-schema.php';
-require __DIR__ . '/phpunit/factories/factory.php';
-require __DIR__ . '/phpunit/factories/documents.php';
 
 require_once dirname( __DIR__ ) . '/includes/autoloader.php';
```

Comment:
Validate the testing library is avalilable

Reply:
Since `require __DIR__ . '/../vendor/autoload.php';` is existing, the PHPUnit will trigger an error about it, so the developer will know he needs to install the composer.